src/test/regress/expected/recursive_dml_with_different_planners_executors.out

```diff
@@ -0,0 +1,87 @@
+CREATE SCHEMA recursive_dml_with_different_planner_executors;
+SET search_path TO recursive_dml_with_different_planner_executors, public;
+CREATE TABLE distributed_table (tenant_id text, dept int, info jsonb);
+SELECT create_distributed_table('distributed_table', 'tenant_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE TABLE second_distributed_table (tenant_id text, dept int, info jsonb);
+SELECT create_distributed_table('second_distributed_table', 'tenant_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE TABLE reference_table (id text, name text);
+SELECT create_reference_table('reference_table');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+INSERT INTO distributed_table SELECT i::text, i % 10, row_to_json(row(i, i*i)) FROM generate_series (0, 100) i;
+INSERT INTO second_distributed_table SELECT i::text, i % 10, row_to_json(row(i, i*i)) FROM generate_series (0, 100) i;
+SET client_min_messages TO DEBUG1;
+-- subquery with router planner
+-- joined with a real-time query
+UPDATE 
+	distributed_table 
+SET dept = foo.dept FROM 
+	(SELECT tenant_id, dept FROM second_distributed_table WHERE dept = 1 ) as foo, 
+	(SELECT tenant_id FROM second_distributed_table WHERE dept IN (1, 2, 3, 4) OFFSET 0) as bar
+	WHERE foo.tenant_id = bar.tenant_id
+	AND distributed_table.tenant_id = bar.tenant_id; 
+DEBUG:  cannot push down this subquery
+DETAIL:  Offset clause is currently unsupported when a subquery references a column from another query
```

Comment:
Minor: Since we `DEBUG1`  all deferred error, it might be confusing as here:`Offset clause is currently unsupported when a subquery references a column from another query`

But, seems hard to fix this.

Reply:
Btw, I personally prefer having the slightly off `DEBUG1` message over not having it at all.